scrapers/javlibrary.yml

```diff
@@ -0,0 +1,29 @@
+name: javlibrary
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.javlibrary.com/
```

Comment:
can we relax the url a bit ?
```yaml
    url:
      - www.javlibrary.com/
```
it seems to me that in some cases javlibrary redirects to http as well and i wouldn't want to manually edit the urls to change to https